core/controllers/editor.py

```diff
@@ -772,3 +774,58 @@ def get(self, exploration_id):
         self.render_json({
             'unresolved_answers': unresolved_answers_with_frequency
         })
+
+
+class LearnerAnswerInfoHandler(EditorHandler):
+    """Handles the learner answer info for an exploration state."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_answer_details
+    def get(self, entity_id, entity_type):
+        """Handles the GET requests for learner answer info for an
+        exploration state.
+        """
+        if not constants.ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE:
+            raise self.PageNotFoundException
+
+        if entity_type == feconf.ENTITY_TYPE_EXPLORATION:
```

Comment:
Need a corresponding else clause.

What about the "elif entity_type == question" case?

Reply:
Should I make it now, or add it further after the project is implemented for question?

Reply:
You should do it now, and make sure that this works for both questions and explorations from the get-go. I thought we discussed that last time when working through milestone 1.

Reply:
Done